tests/integration-tests/tests/efa/test_efa.py

```diff
@@ -215,11 +231,10 @@ def _test_osu_benchmarks_multiple_bandwidth(
         "cat /shared/osu_mbw_mr.out | tail -n +4 | awk '{print $2}' | sort -n | tail -n 1"
     ).stdout
 
-    # Expected bandwidth with 4 NICS:
-    # OMPI 4.1.0: ~330Gbps = 41250MB/s with Placement Group
-    # OMPI 4.1.0: ~252Gbps = 31550MB/s without Placement Group
-    # OMPI 4.0.5: ~95Gbps = 11875MB/s with Placement Group
-    expected_bandwidth = 30000
+    if instance_bandwidth_dict.get(instance) is None:
```

Comment:
minor:
```python

expected_bandwidth = instance_bandwidth_dict.get(instance)
if expected_bandwidth is None:
    pytest.fail(f"Instance {instance} is not valid for multiple bandwidth tests")
```

In this way you don't need to do the get another time.

Reply:
good point will fix